kubel.el

```diff
@@ -955,6 +955,21 @@ P can be a single number or a localhost:container port pair."
     (vterm-send-string command)
     (vterm-send-return))))
 
+(defun kubel-exec-ansi-term-pod ()
+  "Exec into the pod under the cursor -> `ansi-term'."
+  (interactive)
+  (require 'term)
+  (let* ((pod (if (kubel--is-pod-view)
+                  (kubel--get-resource-under-cursor)
+                (kubel--select-resource "Pods")))
+         (containers (kubel--get-containers pod))
+         (container (if (equal (length containers) 1)
+                        (car containers)
+                      (completing-read "Select container: " containers)))
```

Comment:
note for myself: that piece of code is duplicated maybe pieces. should be extracted into a straightforward `kubel--get-container-under-cursor`

Reply:
I add a new func as you suggest. But we duplicate more car and cdr 😆 . Is it good?

Reply:
It's great! Thank you! Saved me a lot of trouble.